assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -286,7 +286,7 @@ function MediaPane(props) {
   useIntersectionEffect(
     refContainerFooter,
     {
-      root: { current: container },
```

Comment:
My view on it: when this code runs `container == null` and it's never re-triggered, even if you add `container` to deps array it will not work because `container` is not in React top-down data flow, you need to have that node in state, so maybe it's better to just use ref for that? @dvoytenko should agree here.